src/Shared/HttpSys/Constants.cs

```diff
@@ -15,8 +15,8 @@ internal static class Constants
         internal const string SchemeDelimiter = "://";
         internal const string DefaultServerAddress = "http://localhost:5000";
 
-        internal static Version V1_0 = new Version(1, 0);
-        internal static Version V1_1 = new Version(1, 1);
-        internal static Version V2 = new Version(2, 0);
+        internal static readonly Version V1_0 = new Version(1, 0);
+        internal static readonly Version V1_1 = new Version(1, 1);
+        internal static readonly Version V2 = new Version(2, 0);
```

Comment:
Are these still used anymore? If not please remove them.

Reply:
They are used in https://github.com/aspnet/AspNetCore/blob/0d412c3775adf2da1a32599f0ded897933f8f36c/src/Shared/HttpSys/RequestProcessing/NativeRequestContext.cs#L157-L174 and https://github.com/aspnet/AspNetCore/blob/0d412c3775adf2da1a32599f0ded897933f8f36c/src/Servers/HttpSys/src/RequestProcessing/Response.cs#L363-L364